website/project/decorators.py

```diff
@@ -289,6 +289,7 @@ def wrapped(*args, **kwargs):
 must_be_contributor = _must_be_contributor_factory(False)
 must_be_contributor_or_public = _must_be_contributor_factory(True)
 must_be_contributor_or_public_but_not_anonymized = _must_be_contributor_factory(include_public=True, include_view_only_anon=False)
+must_be_contributor_and_not_group_member = _must_be_contributor_factory(include_public=True, include_view_only_anon=False, include_groups=False)
```

Comment:
Struggled with either building on existing decorator structure or making a new one. Decided to build on existing with the thought that these won't be around too much longer and might as well just use what we've got for the time being...

Reply:
I agree with you! I think building on the existing decorator is fine